ext/http/01_http.js

```diff
@@ -455,25 +455,39 @@ function upgradeWebSocket(request, options = {}) {
     }
   }
 
-  const response = fromInnerResponse(r, "immutable");
-
   const socket = webidl.createBranded(WebSocket);
   setEventTargetData(socket);
   socket[_server] = true;
-  response[_ws] = socket;
   socket[_idleTimeoutDuration] = options.idleTimeout ?? 120;
   socket[_idleTimeoutTimeout] = null;
 
+  if (inner[_wantsUpgrade]) {
```

Comment:
Inner is an internal object. We don't need to index it using a symbol. You can just use regular property access here.